src/layouts/Footer.jsx

```diff
@@ -1,7 +1,16 @@
 import React from 'react';
+import footerImage from '../assets/images/footer.jpg'; //
 
 const Footer = () => {
-  return <></>;
+  return (
+      <footer style={{ width: '100%', display: 'flex', justifyContent: 'center', alignItems: 'center' }}>
```

Comment:
You should be using Tailwind classes here. We want to try to avoid using inline styling at all the whole project.